daemon/src/maker.rs

```diff
@@ -265,6 +265,8 @@ async fn main() -> Result<()> {
                     cfds,
                 )));
 
+                oracle_actor_address.do_send(oracle::Sync).unwrap();
```

Comment:
`do_send` will block the entire executor thread. Better use `send().await`.

Reply:
woops, thanks! will change :)